wp-content/plugins/core/src/Integrations/Gravity_Forms/Form_Markup.php

```diff
@@ -16,14 +16,14 @@ class Form_Markup {
 	 *
 	 * @filter gform_field_choice_markup_pre_render
 	 *
-	 * @param string $choice_markup
-	 * @param array  $choice
-	 * @param array  $field
-	 * @param string $value
+	 * @param string       $choice_markup
+	 * @param array        $choice
+	 * @param object       $field
+	 * @param array|string $value // GF docs says this should be a string, but both strings and arrays may be returned as of v2.5
 	 *
 	 * @return string
 	 */
-	public function customize_gf_choice_other( string $choice_markup, array $choice, array $field, string $value ): string {
+	public function customize_gf_choice_other( string $choice_markup, array $choice, object $field, $value ): string {
```

Comment:
Looks like we're still using this as an array below, why don't we remove the type (and update the param to be `array|object` in the DocBlock) and then just force cast it as an array?

something like: 

```php
/**
 * @param object|array       $field
 * /
public function customize_gf_choice_other( string $choice_markup, array $choice, $field, $value ): string {
    $field = (array) $field;
    // ...more code
}
```